nim/whip/server.nimble

```diff
@@ -0,0 +1,11 @@
+# Package
+
+version = "0.1.0"
+author = "Willyboar"
+description = "Whip Implementation"
+license = "MIT"
+
+# Dependencies
+
+requires "nim >= 1.0.0"
+requires "whip >= 0.1 & < 1.0"
```

Comment:
better have `whip >= 0.1 & < 0.2`, since we display **0.1 on results (if a 0.3 is published, then we will **NOT** display the right version) with `whip >= 0.1 & < 1.0`